<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlbumExtra</class>
 <widget class="QWidget" name="AlbumExtra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>199</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Quality:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox_shareMode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Embedded Only (&lt;50Kb)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Share Original Images</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Copy Original Images (extra disk space)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Resize Images (&lt; 200Kb)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Resize Images (&lt; 1Mb)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="captionLabel">
     <property name="text">
      <string>Caption:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_Caption"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Photographer:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_Photographer"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Where:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_Where"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>When:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_When"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
